scripts/us_census/pep/monthly_population/download.py

```diff
@@ -0,0 +1,153 @@
+""" A Script to download USA Census PEP monthly population data
+    from the URLS in provided json file.
+    Typical usage:
+    1. python3 download.py

```

Comment:
Please add a README that has a brief description of the data and instructions on how to download and process the data with the commandline flags.

Reply:
Ok We will add README file with detailed steps to execute the scripts.